tests/conftest.py

```
@@ -179,48 +179,45 @@ async def db_engine(
+                        if tables:
+                            for table in reversed(Base.metadata.sorted_tables):
+                                if table.name in tables:
+                                    sync_conn.execute(table.delete())
+                        Base.metadata.create_all(sync_conn)
+
+                    _check_and_drop(conn.sync_connection)
+                    marker_file.touch()
         else:
-            import time
+            # Wait for gw0 to finish initialization
+            root_tmp_dir = tmp_path_factory.getbasetemp() if tmp_path_factory else Path("/tmp")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)